web/src/modules/common/components/__tests__/RemoteContent.spec.js

```diff
@@ -1,8 +1,10 @@
 // @flow
 
 import React from 'react'
-import { shallow } from 'enzyme'
+import { mount, shallow } from 'enzyme'
```

Comment:
Could you solve this with the testing library or test renderer as well? Preferably the first?